nhc

```diff
@@ -433,10 +433,24 @@ function nhcmain_find_rm() {
         NHC_RM="sge"
     fi
 
+    if [[ -d /var/spool/pbs ]]; then
+        NHC_RM="pbspro"
+        return 0
+    fi
+
     # Search PATH for commands
     if type -a -p -f -P pbsnodes >&/dev/null ; then
-        NHC_RM="pbs"
-        return 0
+        # PBSpro prints version to stdout, Torque prints version to stderr
+        VERSION_LINE=( $($(type -a -p -f -P pbsnodes) --version 2>&1) )
```

Comment:
I have tried __very__ hard to eliminate subshells throughout NHC's code, but especially in the driver script.  It would have been far easier to do a bunch of `rpm -q` commands and `--version` and `pkgconfig` and so forth to determine what's installed, but that would've been far less efficient.  So I would prefer to avoid the command executions shown here.  Can you think of any other ways to differentiate PBSpro from TORQUE that wouldn't require executing anything?